src/Krank/Checkers/IssueTracker.hs

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE ApplicativeDo #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Krank.Checkers.IssueTracker (
```

Comment:
module `GithubIssueTracker` perhaps? Or do you think we'll extend this to other tool (such as gitlab) soon?

Reply:
My goal was to put a Gitlab RE in the same file indeed

Reply:
Good !